### PR TITLE
sonarqube: symlink java service wrapper and set defaults

### PR DESCRIPTION
**Pull request by cho-m**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Related to https://github.com/Homebrew/discussions/discussions/3022

### CODE REVIEW

**Comment by SMillerDev** [comment]
I think this is also needed for the LTS now?

**Review by carlocab — DISMISSED** [review]
LGTM

File: Formula/sonarqube.rb
Comment (cho-m) [inline Formula/sonarqube.rb:59]: Before merge, I will probably update regex to avoid greedy capture of whitespace/newlines (`[^/]+`)

Maybe `[^/\s]+`.

Comment (SMillerDev) [reply Formula/sonarqube.rb:59]:
```suggestion
      s.sub! %r{^wrapper\.logfile=.*/([^/\s]+\.log)$}, "wrapper.logfile=#{var}/sonarqube/logs/\\1"
```

**Review by SMillerDev — DISMISSED** [review]

**Comment by cho-m** [comment]
I'll rebase to pick up latest version of `sonarqube` and then will drop `do not merge` label.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**
